lib/logstash/outputs/loggly.rb

```diff
@@ -78,6 +83,12 @@ class LogStash::Outputs::Loggly < LogStash::Outputs::Base
   # Proxy Password
   config :proxy_password, :validate => :password, :default => ""
 
+  # This plugin uses the bulk index api for improved indexing performance.
+  # This setting lets you ensure you don't send too many events at a time.
+  # See the bulk API documentation to estimate an appropriate amount of events,
+  # based on your expected event size.
+  # https://www.loggly.com/docs/http-bulk-endpoint/
+  config :flush_size, :validate => :number, :default => 100, :required => true
```

Comment:
if it makes things simpler I'd be totally ok with not providing this cap option and just let `pipeline.batch.size` and `pipeline.batch.interval` decide the event count (which the user can configure if necessary)

Reply:
I like the idea. Since we discussed splitting the payload on 413, might as well go straight to the approach of checking size as we encode, and remove this now unnecessary attribute.

I will still support splitting the payload on 413, in case their limits change from 5Mb/call.